File: scripts/py-compile-suite/inputs/21_slicing_2d.m
% 2D slicing and reshaping
A = reshape(1:24, [4, 6]);
row1 = A(1, :);
col2 = A(:, 2);
submat = A(1:2, 3:5);
last_row = A(end, :);
last_col = A(:, end);
A_flat = A(:);

File: scripts/py-compile-suite/outputs/21_slicing_2d.py
import numpy as np

# 2D slicing and reshaping
A = np.arange(1, 24 + 1).reshape([4, 6])
row1 = A[0, :]
col2 = A[:, 1]
submat = A[0:2, 2:5]
last_row = A[-1, :]
last_col = A[:, -1]
A_flat = A.flatten(order="F")
